libass/ass_utils.c

```diff
@@ -511,3 +511,26 @@ ASS_Style *lookup_style_strict(ASS_Track *track, char *name, size_t len)
     return NULL;
 }
 
+#if defined(_WIN32) && !defined(__CYGWIN__)
+wchar_t* ass_convert_to_utf16le(const char* data, UINT in_code_page) {
+    int len_utf16le = MultiByteToWideChar(in_code_page, MB_ERR_INVALID_CHARS, data, -1, NULL, 0);
+    if (!len_utf16le) return NULL;
```

Comment:
Per libass' style, control flow statements like `break`, `return`, etc should always be placed on their own line.